ci/plugins/mzcompose/hooks/pre-exit

```diff
@@ -38,6 +62,22 @@ buildkite-agent artifact upload ps-aux.log
 docker ps -a --no-trunc > docker-ps-a.log
 buildkite-agent artifact upload docker-ps-a.log
 
+# Test
+known_issue 16778
+
+# Test 2
+known_issue 17616
+
+if grep "error: missing record 0: Record {" services.log; then
+  known_issue 16778
+fi
+if grep "thread 'coordinator' panicked at 'can't persist timestamp: Error { kind: Stash(StashError { inner: Other(\"unexpected lower\") }) }'" services.log; then
+  known_issue 16927
+fi
+if grep "thread 'coordinator' panicked at 'reference to absent collection: CollectionMissing(User(343))'" services.log; then
```

Comment:
Just CollectionMissing would be enough, the number in User() will change across invocations.